kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/api/common/AnyLocalRef.java

```diff
@@ -44,6 +46,7 @@ public AnyLocalRefBuilder edit() {
     private String group;
 
     @Override
+    @Nullable
```

Comment:
In another PR we should push these up to the base class, since they can be null in `AnyLocalRef`.